test/resources/webhook.test.js

```diff
@@ -82,4 +83,43 @@ describe('Webhook Resource', function () {
 
     expect(deletedWebhook).to.be.an.instanceOf(this.easypost.Webhook);
   });
+
+  it('validates a webhook secret', function () {
+    const webhookSecret = 'sécret';
+    const expectedHmacSignature =
+      'hmac-sha256-hex=e93977c8ccb20363d51a62b3fe1fc402b7829be1152da9e88cf9e8d07115a46b';
```

Comment:
Not: if this never changes, should we store it as a fixture?

Reply:
Typically fixtures are for input data, not assertions/expectations. There would also not be a very concise way of saying "this signature belongs to this data over here"